dist-persist/wbstack/src/Settings/LocalSettings.php

```diff
@@ -44,6 +44,16 @@
     $wgShowExceptionDetails = true;
 }
 
+// profiling
+// requires docker image to be built with dependencies
+if( isset ( $_SERVER['HTTP_X_WBSTACK_MW_PROFILING'] ) ) {
```

Comment:
This should maybe, probably be hidden behind something more.

Reply:
I guess we don't want users of the public service to be able to enable this with a header (even though the actual profiling code won't be baked into their docker image). We could add another environment variable and a check here.

Alternatively we could put this in a separate file and just copy it in for certain builds of the image

Reply:
> Alternatively we could put this in a separate file and just copy it in for certain builds of the image

+1 sounds like a good approach to me!

Reply:
I'm not sure how to conditionally copy this new localsettings file into the container without having to shuffle a bunch of stuff around in the build context which seems to be the suggested way of doing that. COPY can take files that sometimes doesn't exist.

Feels however like this just makes everything more complex and fragile, I think just setting a new value like `ENV MW_ENABLE_PROFILING_HEADER=${INSTALL_PROFILING_DEPS}
` and look that this is the same expected value the install_profiling.sh script expects.

Reply:
something like https://github.com/wbstack/mediawiki/pull/252/commits/f4ce15e66cdd22c82ad6dabe03f2592546e3ba74